x/wasm/handler.go

```diff
@@ -95,18 +97,19 @@ func handleInstantiate(ctx sdk.Context, k Keeper, msg *MsgInstantiateContract) (
 		return nil, err
 	}
 
-	events := filterMessageEvents(ctx.EventManager())
-	ourEvent := sdk.NewEvent(
+	events := filteredMessageEvents(ctx.EventManager())
```

Comment:
Looks like a good update here to combine the events in the same way with different types